teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] The assignment of `parent_permission` in the `super` method is a little different since it uses a colon (:)

  ```
  super(age, name, parent_permission: parent_permission)
  ```

Reply:
@julie-ify  Thanks for that reminder